cmd/export.go

```diff
@@ -157,18 +160,38 @@ To export everything about a dataset, use the --dataset flag.`,
 			}
 
 			metaPath := filepath.Join(path, dsfs.PackageFileMeta.Filename())
-			mdBytes, err := json.MarshalIndent(md, "", "  ")
+			mdBytes := []byte{}
```

Comment:
This should be `var mdBytes []byte`. 

`mdBytes := []byte{}` allocates a byte slice with a length & cap of 0 and assigns it to `mdBytes`. This initial allocation is never used b/c we're only ever assigning `mdBytes` to other newly-allocated byte-slices below. 

`var mdBytes []byte` declares "I need memory for a slice of bytes", but does no actual byte-slice allocation.